README.md

```diff
@@ -1,16 +1,167 @@
-| `develop` | ![run-sdk-tests](https://github.com/finbourne/lusid-sdk-python/workflows/run-sdk-tests/badge.svg?branch=develop) |
+<a href="https://www.lusid.com/app/signup"><img src="https://content.finbourne.com/LUSID_repo.png" alt="LUSID_by_Finbourne"></a>
 
-# LUSID<sup>®</sup> Python SDK
+## Installation
 
 The PyPi package for the LUSID SDK can installed using the following:
 
 ```
-$ pip install lusid-sdk
+pip install lusid-sdk
+```
+
+We publish two versions of the Python SDK:
+
+* **lusid-sdk-python (this one) - supports `Production` and `Early Access` API endpoints**
+* [lusid-sdk-python-preview](https://github.com/finbourne/lusid-sdk-python-preview) - supports `Production`, `Early Access`, `Beta` and `Experimental` API endpoints.
+
+For more details on API endpoint categories, see [What is the LUSID feature release lifecycle?](https://support.lusid.com/knowledgebase/article/KA-01786/en-us).
+To find out which category an API endpoint falls into, see [LUSID API Documentation](https://www.lusid.com/api/swagger/index.html).
+
+> If you install both `lusid-sdk-python` and `lusid-sdk-python-preview`, `import lusid` will refer to `lusid-sdk-python`. 
+
+## Usage
+
+### Create API Client
+
+First, import the following LUSID modules:
+
+
+```python
+import lusid
+import lusid.models as models
+from lusid.utilities import ApiConfigurationLoader
+```
+
+And construct the API factory:
+
+```python
+secrets_file_path = "/path/to/secrets.json"
+config = ApiConfigurationLoader.load(secrets_file_path)
+api_factory = lusid.utilities.ApiClientFactory(
+    token=lusid.utilities.RefreshingToken(config),
+    api_secrets_filename=secrets_file_path
+)
+```
+
+> You will need to create a secrets file containing your LUSID credentials. For instructions on how to construct this file, see [Getting started with the LUSID API and SDKs](https://support.lusid.com/knowledgebase/article/KA-01916/)
+
+Now that the API client is ready, you are ready to use the various API endpoints.
+You can list all the API endpoints by running the following: 
+
+```python
+[api for api in dir(lusid.api) if "API" in api]
+```
+
+An API endpoint can be constructed by calling `api_factory.build(lusid.api.<className>)` for any of the returned classes.
+
+### Code Samples
+
+Before running the examples, import the following modules:
+
+```python
+import uuid
+import datetime
+import pytz
 ```
 
-For further documentation on building the SDK, running the tutorials and using the SDK please see the [wiki](https://github.com/finbourne/lusid-sdk-python/wiki).
+The examples below should be run in order, as they assume that the preceding code has been executed.
+
+> For a Python script containing all the sample code, see [sdk/examples/hello_world.py](sdk/examples/hello_world.py).
```

Comment:
if we are putting it in a new `examples` folder (which is fine), we should probably exclude if from the distributed package (see https://github.com/mneedham/lusid-sdk-python/blob/readme-refactoring/sdk/setup.py#L58)

Reply:
Done!